docs/app/pages/components/sections/date-time-picker/time-picker/snippets/app.html

```diff
@@ -10,67 +10,42 @@
-                            <label for="maxMinute">max minute</label>
-                            <ux-number-picker [(value)]="maxMinute" min="0" max="59" (valueChange)="maxTime.setMinutes($event)"></ux-number-picker>
-                        </div>
+                        <label for="myMaxTime">max time:</label>
+                        <timepicker [(ngModel)]="max" [min]="min"></timepicker>
                     </div>
                 </div>
 
                 <div class="row uxd-customize-row">
                     <div class="col-md-6 col-sm-6">
-                        <div>
-                            <label for="hstep">hours step</label>
-                            <ux-select [(value)]="hstep" [options]="options.hstep"></ux-select>
-                        </div>
+                        <label for="hourStep">hours step</label>
+                        <ux-number-picker [(value)]="hourStep" min="1" max="22" (valueChange)="hourStep=($event)"></ux-number-picker>
                     </div>
                     <div class="col-md-6 col-sm-6">
-                        <div>
-                            <label for="mstep">minutes step</label>
-                            <ux-select [(value)]="mstep" [options]="options.mstep"></ux-select>
-                        </div>
+                        <label for="minuteStep">minutes step</label>
+                        <ux-number-picker [(value)]="minuteStep" min="1" max="58" (valueChange)="minuteStep=($event)"></ux-number-picker>
                     </div>
                 </div>
 
                 <div class="row m-t-md">
                     <div class="col-md-12 col-sm-12">
-                        <div>
-                            <ux-checkbox [(value)]="checkModel.showMeridian" (click)="ismeridian = !ismeridian">12H / 24H</ux-checkbox>
-                        </div>
+                        <ux-checkbox [(value)]="checkModel.showMeridian" (click)="showMeridian = !showMeridian">12H / 24H</ux-checkbox>
                     </div>
                 </div>
-
                 <div class="row m-t-md">
                     <div class="col-md-12 col-sm-12">
-                        <div>
-                            <ux-checkbox [(value)]="checkModel.showSeconds" (click)="showSec = !showSec">show / hide seconds</ux-checkbox>
-                        </div>
+                        <ux-checkbox [(value)]="checkModel.showSeconds" (click)="showSeconds = !showSeconds">show / hide seconds</ux-checkbox>
                     </div>
                 </div>
 
                 <div class="row m-t-md">
                     <div class="col-md-12 col-sm-12">
                         <label>disable:</label>
                         <div>
-                            <ux-checkbox [(value)]="checkModel.readonlyInput" (click)="isEnabled=!isEnabled">input</ux-checkbox>
+                            <ux-checkbox [(value)]="checkModel.readonlyInput" (click)="readonlyInput=!readonlyInput">input</ux-checkbox>
```

Comment:
I don't think you need a (click) event handler here - [(value)] means that it's 2-way bound to the property that you specified. Therefore angular will keep that property in sync. (Same for all other controls here.)

If it would help, you can refer to the implementation of one of the other demos, e.g. tag input:
https://github.com/UXAspects/UXAspects/blob/develop/docs/app/pages/components/sections/input-controls/tags/tags.component.html#L40
https://github.com/UXAspects/UXAspects/blob/develop/docs/app/pages/components/sections/input-controls/tags/tags.component.ts

Reply:
Check boxes stop working when I try to remove (click)

Reply:
Instead of using checkModel.readonlyInput, try creating a readonlyInput property directly on the component and use that as the binding.